utils/spanutils/spanutils.go

```diff
@@ -83,7 +83,8 @@ func (span *tracingSpan) SetError(msg string) {
 		return
 	}
 	if msg != "" {
-		span.openSpan.SetTag("error", msg)
+		span.openSpan.SetTag("error", true)
+		span.openSpan.SetTag("error.msg", msg)
```

Comment:
according to [Jaeger document](https://opentracing.io/docs/overview/tags-logs-baggage/), they don't mention anything about the size difference between Tag and Log. I think based on our needs, Tag would be better than using Log. It looks good for me.

Reply:
I will create another PR to use SetError in every endpoint call.